application/main.tf

```diff
@@ -88,3 +88,23 @@ data "aws_ami" "centos" {
     values = ["ebs"]
   }
 }
+
+data "aws_ami" "centos_wls" {
+  owners      = ["895523100917"]
+  most_recent = true
+
+  filter {
+    name   = "name"
+    values = ["HMPPS Delius-Core Weblogic Managed adminAndManagedAmis *"]
```

Comment:
use master please, this ami is the same thing and may be a bit faulty

Reply:
done